spec/warden/jwt_auth/header_parser_spec.rb

```diff
@@ -55,5 +55,18 @@
 
       expect(headers).to eq('Authorization' => 'Bearer 123')
     end
+
+
+    describe 'custom token header' do
+      it 'returns configured token_header' do
+        allow(Warden::JWTAuth.config).to receive(:token_header).and_return('Test_Authorization')
+
+        headers = {}
+
+        headers = described_class.to_headers(headers, '123')
+
+        expect(headers).to eq('Test_Authorization' => 'Bearer 123')
+      end
+    end
```

Comment:
No need. The test above already tests it and we're testing it can be changed in the `env_helper` test file.